config/install/core.entity_view_display.node.landing_page_lb.default.yml

```diff
@@ -15,6 +21,8 @@ dependencies:
     - bootstrap_layout_builder
     - layout_builder
     - layout_builder_restrictions
+    - metatag
+    - openy_gtranslate
```

Comment:
new dependency should be reflected in info.yml and composer.json

Reply:
See https://github.com/YCloudYUSA/y_lb/pull/122/commits/e5fc78982fe873782c8232ca649cf2899fe71a88 and https://github.com/YCloudYUSA/y_lb/pull/122/commits/cc037e61dd5b4c3029b104f839dd7737bc6518ad @froboy